src/content/en/2020/fonts.md

```diff
@@ -22,3 +22,278 @@ queries: 04_Fonts
+
+{{ figure_markup(
+  image="fonts-popular-typefaces.png",
+  caption="Popular typefaces.",
+  description="A bar chart showing the 10 most popular web fonts, starting with Font Awesome (35% of desktop and mobile), Open Sans (23% desktop and 25% of mobile), Roboto (17% desktop and 23% mobile), Glyphicons Halflings (16% for both), Lato (11% for both), Montserrat (19% for both), Font Awesome 5 Brands (19% for both), Font Awesome 5 Free (9% desktop and 8% mobile), Raleway (6% for both), and dashicons (6% for both).",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT2Q4hcDGGdclJH2ym0Pp_f8JWvYur_OQFQNkuScJyO7_ZCR1KPZsewL-mEZhxcuRFcde_Mxio8z_8P/pubchart?oid=551344676&format=interactive",
+  sheets_gid="179750099",
+  sql_file="04_11a.popular_typeface.sql",
+  width="600",
+  height="520"
+  )
+}}
+
+A note of caution, in determining the most popular fonts you can get different results depending on measurement methodology. The chart above is based on counting the number of pages that include an `@font-face` rule referencing the named font. That counts multiple styles only once, which arguably weights in favor of single-style fonts.
+
+## Color fonts
+
+Color fonts, in one form or other, are supported by most modern browsers, but usage is still close to nonexistent (a total of 755 pages total, the majority of which are in a format that is not widely supported in popular browsers). No doubt part of the problem is the diversity of formats, in fact 4 in widespread use. These come in bitmap and vector flavors. The two bitmap formats are technologically very similar, but SBIX (originally a proprietary Apple format) is not supported in Firefox, while CBDT/CBLC is not supported in Safari.
+
+The COLR vector format is supported on all major modern browsers, but only fairly recently. The fourth format is embedding SVG in OpenType (not to be confused with SVG fonts), but not supported in Chrome. One drawback of SVG in OpenType is lack of support for font variations, an increasingly important aspect of modern Web design. For this reason, the COLR format is likely to prevail, particularly as support for gradients and clipping is being developed for a future version of COLR is.
+
+One reason for the poor support of color fonts on the web is that the colors have to be baked into the font files themselves. If you use the same typeface with three different color combinations, near-identical files have to be downloaded three times, and changng a color means reaching for a font editor.
+
+While there is a feature in CSS to [override or replace the color palettes in fonts](https://drafts.csswg.org/css-fonts-4/#font-palette-values), this has not yet been implemented in browsers which certainly holds back the ease of deploying color web fonts.
+
+Generally the bitmap format has larger files and doesn’t scale as cleanly to larger sizes, but COLR only offers flat color shading (though work is ongoing to offer richer shading options in a future version).
```

Comment:
Already been said above, with

> particularly as support for gradients and clipping is being developed for a future version of COLR is.

Maybe leave out this paragraph altogether and add the large size argument the the paragraph discussing CBDT and sbix?